Work14/Program.cs

```diff
@@ -0,0 +1,263 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Work4
+{
+    enum NumberToWord
```

Comment:
use EN name!

Reply:
I trained on my daughter (8 years old), she played for 2 days, won 1000 once), ok